src/openms/source/FILTERING/ID/IDFilter.cpp

```diff
@@ -782,6 +782,34 @@ namespace OpenMS
     }
   }
 
+  void IDFilter::keepNBestSpectra(std::vector<PeptideIdentification>& spectra, Size n)
```

Comment:
Why call this `spectra` and not `IDs`?
Technically you do not deal with spectra, but I guess ID might be mistaken for PSM, i.e. PeptideHit?
Any other suggestions?

Reply:
puh yeah I always think about spectra and PSMs in that context. What you want here is to retain the best n spectra and their PSMs. In the new data structure this would be InputItem but isn't much more telling either

Reply:
Will rename